keyboards/hfdkb/ac001/info.json

```
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "ac001",
+    "url": "",
+    "maintainer": "qmk",
+    "layouts": {
```

Note, I have not included a vendor ID here, because 0x3528 is officially assigned to Klein Tools. 
https://www.the-sz.com/products/usbid/index.php?v=0x3528&p=&n=

```suggestion
    "usb": {
        "vid": "",
        "pid": "0x2330",
        "device_version": "0.0.1"
    },
    "layouts": {
```